data/channels.csv

```diff
@@ -18636,7 +18636,8 @@ PremiumStories.it,Premium Stories,,,,IT,,,c/IT,ita,,FALSE,2015-06-23,2022-01-10,
 PremiumStoriesHD.it,Premium Stories HD,,,,IT,,,c/IT,ita,,FALSE,2018-06-04,2022-01-10,,,https://upload.wikimedia.org/wikipedia/it/9/9a/Premium_Stories_Logo_2015.png
 PremiumStoriesPlus24.it,Premium Stories +24,,,,IT,,,c/IT,ita,,FALSE,2016-03-01,2017-09-04,,,https://upload.wikimedia.org/wikipedia/it/9/9a/Premium_Stories_Logo_2015.png
 PresenceTVChannel.et,Presence TV Channel,,,,ET,,,c/ET,amh,,FALSE,,,,,https://www.lyngsat.com/logo/tv/pp/presence-tv-channel-et.png
-PressTV.ir,Press TV,,,Islamic Republic of Iran Broadcasting,IR,,,c/IR,eng,news,FALSE,2007-07-08,,,https://www.presstv.ir/,https://upload.wikimedia.org/wikipedia/commons/thumb/6/63/Press_Tv_logo.svg/512px-Press_Tv_logo.svg.png
+PressTV.ir,Press TV,,,Islamic Republic of Iran Broadcasting,IR,,,c/IR,eng,news,FALSE,2007-07-08,,,https://www.presstv.ir/,https://www.presstv.ir/custom/images/site/LOGO.gif

```

Comment:
```suggestion
PressTV.ir,Press TV,,,Islamic Republic of Iran Broadcasting,IR,,,c/IR,eng,news,FALSE,2007-07-08,,,https://www.presstv.ir/,https://french.presstv.ir/img/logo.png
```
No need for two separate logo links when they're the same.

Reply:
But what was wrong with the old logo?

https://upload.wikimedia.org/wikipedia/commons/thumb/6/63/Press_Tv_logo.svg/512px-Press_Tv_logo.svg.png

Reply:
Ah, wrong one I've chosen.